utils/gha-runners/createTestconfig.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright 2014-2022, Intel Corporation
+
+#
+# createTestconfig.sh - Script for creating testconfig files for test excution.
+#
+
+# Default location for testconfig.sh
+CONF_PATH="src/test"
+MOUNT_POINT=${PMDK_MOUNT_POINT:-"/mnt/pmem0"}
+NON_PMEM_DIR="/dev/shm"
+
+# Create config file for unittests.
+# We are using ndctl command to gather information about devdaxes, in form known from namespace configuration.
+cat >${CONF_PATH}/testconfig.sh <<EOL
+# main & local
+PMEM_FS_DIR=${MOUNT_POINT}
+NON_PMEM_FS_DIR=${NON_PMEM_DIR}
+DEVICE_DAX_PATH=($(ndctl list -X | jq -r '.[].daxregion.devices[].chardev' | awk '$0="/dev/"$0' | paste -sd' '))
+KEEP_GOING=y
+TM=60m
```

Comment:
This is not timeout variable. Please look at https://github.com/pmem/pmdk/blob/master/src/test/testconfig.sh.example